cmd/vic-machine/configure/configure.go

```diff
@@ -184,6 +257,10 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 			v.Network.Nameservers = n.ExecutorConfig.Networks[k].Network.Nameservers
 		}
 	}
+	o.HostCertificate = n.HostCertificate
```

Comment:
No, because remember later on down the line, we omit any `nil` members and don't overwrite the existing member (this is the reason `--no-tls` is not part of this PR)